tests/unit/files/metadata_default_namespaces.xml

```diff
@@ -0,0 +1,25 @@
+      </KeyInfo>
+    </KeyDescriptor>
+    <KeyDescriptor use="encryption">
+      <KeyInfo xmlns="http://www.w3.org/2000/09/xmldsig#">
+        <X509Data xmlns="http://www.w3.org/2000/09/xmldsig#">
+          <X509Certificate xmlns="http://www.w3.org/2000/09/xmldsig#">MIIFOjCCAyKgAwIBAgIUCWcJxrX+RK7+1rxXeiOk2Pef3NMwDQYJKoZIhvcNAQELBQAwHjEcMBoGA1UEAwwTc2FtbC5jYW5vbmljYWwudGVzdDAeFw0yMzA4MjYxNDQ5NDZaFw0zMzA4MjMxNDQ5NDZaMB4xHDAaBgNVBAMME3NhbWwuY2Fub25pY2FsLnRlc3QwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQCvRliUlWDkDSCrRz/H5oTpRfIhMKJ/EwE6aWVue+GxdXivXRSBQFnPquE2XDcPPwEqsmjJ58pZ1D3hhPHQtyY9lhTcBP7TbRo2lx4aT0smCUellwpw4AqqARcPQvBPJ5s+sz4mRk7t8zV0eKjLTvvq8/qWKi1hHW9wJb9hD8Ie3Y3BbI5cc3+K9Nhqq2qNg7Pkmo7dvCDP3F8v+0gDRM6XuNhVq97EdjzUcXbHjj+CbjCeQKxBnhgfdHZEwbvqRv8YuWvJ2PmNUqFgjQRi2x0kXJyJW1OPaW83DiSvkqOaFb1rSt8/tK3ZdQ9YcoXf8bdJyRgrfS6YIJf+1o+ckty7rUEk+LhXH2OGg0rA6MIbTiFrMw4PVTRHJac0q8w4puQzTTKcCTBkj7Wi2sEKcOeauDYueLwGOF7pY4NL1em7xz7zYTsFYcDG5qnjdQAJOLCEJRKWoQezaLW4RTTbCJ0sa7U/lNtym45qwhkM5zR4kyLHV0U0XKTRrD94HslVLPZrxX9MXEos+ys6wWcxovPf5IMg4e3iMqxuwJmIOEMXTM/0GNjlNeovvUvbKz+nA1lIYOnf8BmnNoBUvJoIe02igU7+NBRvlqByYByJBz90JXa0Knh1kK+xCjo4deRtbe1JzWEEJZ+fB58SEn2C/ulJbngnb+uh4x1UgvbkWHH9wQIDAQABo3AwbjAdBgNVHQ4EFgQUVMab5jNo59oiOT2G91sxDCiZPuIwHwYDVR0jBBgwFoAUVMab5jNo59oiOT2G91sxDCiZPuIwDAYDVR0TAQH/BAIwADAeBgNVHREEFzAVghNzYW1sLmNhbm9uaWNhbC50ZXN0MA0GCSqGSIb3DQEBCwUAA4ICAQB5tLn9y3E/MciDkFSh1/PYAyNjs+zGxpm/KH95pKyVWeqFSt09+FnOVI/RUVsx7+FV3++gKfR5flsDbWhmtNAWvujyls7DFuWpf0pDfkrzaul3K1Fu2wF+xVklB+YYVDlP2LYrAdONiVq0Dd5a69tniwTDLjaKhhNJqOTrg/OywmDBa/Ym7kyih5oWP7fdoK9MBeFRrCJsD/831ax+LtNMhp+wgM7cmKJOLqNBbqRDhyPEk9XSK5gJdP1ty/2IEfktsGiPrjR6Wl2WvUj6BlCBFHh7aM4IYjuRo/XU+3yyEb2hvlEYT3rDaNa4ZBtGub6K9/M78CBPULaKlGWAVmdtXH8+04YRxWpyfP0XPPUzkzqAvLs6WEBeD1zEN/+1AtyGEAc0/e7LKwblOSXQC/lkjhiwpIF1UV24wHHDsOoMMvcAF+pJLZkPDKqYAktohhQjtUOOyQ1QxodY6F6ENfimlS0cDP7ngmEswe50VJ52otZvmq1n5aiyeF1hCcIQ8IqeLVrpIMcEA5GTMZgrbnUsjePNoD6kNuXzYrs4+/UIQfl8R51cuONh5jvLVT8SUR/6AYyVeUsK2VjwHFhIR5BRPTGxPsURbOWsjbzsVKFEiCfiG2ciUaexjZClXhRR0wpLqHu5luiR6FMXfE7BrjeJ/VTU37n9580tzlVKk1AA3Q==</X509Certificate>
+        </X509Data>
+      </KeyInfo>
+      <EncryptionMethod Algorithm="http://www.w3.org/2001/04/xmlenc#aes128-cbc"></EncryptionMethod>
+      <EncryptionMethod Algorithm="http://www.w3.org/2001/04/xmlenc#aes192-cbc"></EncryptionMethod>
+      <EncryptionMethod Algorithm="http://www.w3.org/2001/04/xmlenc#aes256-cbc"></EncryptionMethod>
+      <EncryptionMethod Algorithm="http://www.w3.org/2001/04/xmlenc#rsa-oaep-mgf1p"></EncryptionMethod>
+    </KeyDescriptor>
+    <NameIDFormat>urn:oasis:names:tc:SAML:2.0:nameid-format:transient</NameIDFormat>
+    <SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://saml.canonical.test/sso"></SingleSignOnService>
+    <SingleSignOnService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://saml.canonical.test/sso"></SingleSignOnService>
+  </IDPSSODescriptor>
+</EntityDescriptor>
```

Comment:
I think this file is missing a final new line

Reply:
Yes, but this file is returned from the IdP server, so I just want to keep it as is.